include/alpaka/dev/Traits.hpp

```diff
@@ -136,6 +143,21 @@ namespace alpaka
                     dev);
         }
 
+        //-----------------------------------------------------------------------------
+        //! \return The warp size on the device in number of threads.
+        template<
+            typename TDev>
+        ALPAKA_FN_HOST auto getWarpSize(
+            TDev const & dev)
+        -> std::size_t
```

Comment:
Suggestion: we should not enforce a 64bit type here and let the compiler select the return type by looking to the implementation of the trait..

Reply:
This is not kernel-side, so does it matter?

Reply:
I think for a common use case not, therefore it is only a suggestion.